src/api/handlers.js

```diff
@@ -0,0 +1,25 @@
+var req_validations = require("./validations.js").request_validations()
+
+module.exports.request_handlers = function() {
+	return {
+		get_experiment: function(experiment_id) {
```

Comment:
@anmolbabu @mik-dass  Can you suggest what Handler should I add? I think get_experiment is manager-api specific.

Reply:
Please have a look at https://github.com/fabric8-hdd/measure-api/pull/1#issuecomment-359366518
I think ingest data or add_measure or something on those lines would be appropriate here

Reply:
I think handle_stats_ingestion or something on those lines.